site/content/docs/5.1/forms/quantity-selector.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+title: Quantity selector
+description: Form element used to select a number.
+group: forms
+toc: true
+---
+
+## Examples
+
+Quantity selector is a form element used to select a number. For proper styling, use one of the two **required** contextual classes (e.g., `.quantity-selector-large`).
```

Comment:
If it has a 40px height, it is rather the normal display no? that we should use without any classes (`.quantity-selector-large` or `quantity-selector-l`).
An so the other one would be `quantity-selector-sm` (same naming than for the buttons or alerts for example).

Reply:
Done